molgenis-app/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM tomcat:8.5-jre8
```

Comment:
MOLGENIS requires Oracle JDK, not OpenJDK.

Reply:
This is problematic.
Creating images based on the oracle jdk is dangerously close to redistributing, which the license forbids.

I ran into issues where molgenis would not start using OpenJDK a couple of years ago when I tried to run molgenis on ubuntu, but the current version appears to be doing fine. Oracle and OpenJDK are converging, so perhaps the issues we had are simply no longer there.

For laughs, I created an image `FROM tomcat:8.5.32-jre10-slim` with current war in it and it runs nicely (and blazing fast)